vdyp-sindex/src/main/Sindxdll.java

```diff
@@ -0,0 +1,3748 @@
+import java.util.NoSuchElementException;
+
+public class Sindxdll{
+/**

```

Comment:
Javadoc for a class should preceed the class but be at the top inside of it.  The history shouldn't be part of the Javadoc but a separate normal comment.